indico/modules/receipts/util.py

```diff
@@ -145,22 +148,58 @@ def compile_jinja_code(code: str, template_context: dict, *, use_stack: bool = F
         raise UnprocessableEntity(e)
 
 
-def sandboxed_url_fetcher(event: Event, allow_event_logo: bool = False) -> t.Callable[[str], dict]:
+def sandboxed_url_fetcher(event: Event, allow_event_images: bool = False) -> t.Callable[[str], dict]:
     """Fetch also "event-local" URLs.
 
     More info on fetchers: https://doc.courtbouillon.org/weasyprint/stable/first_steps.html#url-fetchers
     """
     allow_external_urls = receipts_settings.get('allow_external_urls')
 
     def _fetcher(url: str) -> dict:
-        if allow_event_logo and url == 'event://logo':
-            return {
-                'mime_type': event.logo_metadata['content_type'],
-                'string': event.logo
-            }
+        url_data = urlparse(url)
+        if allow_event_images and url_data.scheme == 'event':
+            if url_data.netloc == 'logo':
+                return {
+                    'mime_type': event.logo_metadata['content_type'],
+                    'string': event.logo
+                }
+            if url_data.netloc == 'placeholder':
+                placeholder_image = Path(current_app.root_path) / 'web' / 'static' / 'images' / 'placeholder_image.svg'
```

Comment:
Is that image self-made (ie no licensing issues / attribution requirements) or based on an icon set we're already using?

Reply:
the background is from freesvg, which doesn't have any attribution requirements AFAIK..
the icon is the image icon from fontawesome/SUI, so again no issues there

Reply:
https://freesvg.org/? that's indeed CC0 / public domain --> fine

Reply:
yep. this image in particular - https://freesvg.org/graphic-placeholder